gateway/sds_gateway/users/views.py

```diff
@@ -307,35 +310,54 @@ class GroupCapturesView(LoginRequiredMixin, FormSearchMixin, TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        base_dir = f"/files/{self.request.user.email}"
```

Comment:
for consistency, you can probably call `utils.sds_files.sanitize_path_rel_to_user()` here, something like:

```py
base_dir = sanitize_path_rel_to_user(unsafe_path="/", request=self.request)`
```